File: messages/src/main/java/com/thundashop/zauiactivity/dto/OctoBooking.java
package com.thundashop.zauiactivity.dto;

import java.util.List;

import lombok.Data;

@Data
public class OctoBooking {
    private String id;
    private String uuid;
    private String supplierReference;
    private String status;
    private String resellerReference;
    private String utcCreatedAt;
    private String utcUpdatedAt;
    private String utcExpiresAt;
    private String utcRedeemedAt;
    private String utcConfirmedAt;
    private String productId;
    private String freesale;
    private String availabilityId;
    private String notes;
    private String optionId;
    private OctoProductAvailability availability;
    private Contact contact;
    private List<String> deliveryMethods = null;
    private List<Ticket> voucher = null;
    private List<UnitItemOnBooking> unitItems = null;
    private Cancellation cancellation;
    private Pricing pricing = null;
}

@Data
class DeliveryOption {
    private String deliveryFormat;
    private String deliveryValue;
}

@Data
class Ticket {
    private String redemptionMethod;
    private String utcDeliveredAt;
    private String utcRedeemedAt;
    private List<DeliveryOption> deliveryOptions = null;
}

@Data
class Cancellation {
    private Boolean cancelled;
    private String utcCancelledAt;
}

File: com.getshop.client/events/zauiactivity/dto/OctoBooking.php
<?php
class zauiactivity_dto_OctoBooking {
	/** @var String */
	public $id;

	/** @var String */
	public $uuid;

	/** @var String */
	public $supplierReference;

	/** @var String */
	public $status;

	/** @var String */
	public $resellerReference;

	/** @var String */
	public $utcCreatedAt;

	/** @var String */
	public $utcUpdatedAt;

	/** @var String */
	public $utcExpiresAt;

	/** @var String */
	public $utcRedeemedAt;

	/** @var String */
	public $utcConfirmedAt;

	/** @var String */
	public $productId;

	/** @var String */
	public $freesale;

	/** @var String */
	public $availabilityId;

	/** @var String */
	public $notes;

	/** @var String */
	public $optionId;

	/** @var zauiactivity_dto_OctoProductAvailability */
	public $availability;

	/** @var zauiactivity_dto_Contact */
	public $contact;

	/** @var String */
	public $deliveryMethods;

	/** @var class com_thundashop_zauiactivity_dto_Ticket[] */
	public $voucher;

	/** @var zauiactivity_dto_UnitItemOnBooking[] */
	public $unitItems;

	/** @var zauiactivity_dto_Cancellation */
	public $cancellation;

	/** @var zauiactivity_dto_Pricing */
	public $pricing;

}
?>